src/graphql/query/flssLoansQuery.graphql

```diff
@@ -1,11 +1,15 @@
 #import '../fragments/loanCardFields.graphql'
 
 query flssLoanData(
-  $filterObject: [FundraisingLoanSearchFilterInput!]
+  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
   $imgDefaultSize: String = "w480h360"
   $imgRetinaSize: String = "w960h720"
+  $limit: Int = 50
```

Comment:
This is too high... Two reasons, 1. we couldn't show that many loans on a single screen ( currenly we might show 9 on a big screen), 2. we would never want to try and ingest that many loans into a client at one time as all the extra data in loanCardFields has to resolve from graphql too which would create a massive data payload.

Reply:
thanks for the insight, Matt! You got to the comment box before my next push went through 😄 
you're 💨  fast!

Reply:
appreciate the eyes and the explanation!